app/halgo/Halgo/CLA/Flag.hs

```diff
@@ -0,0 +1,40 @@
+-- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+-- | Flags for commands
+module Halgo.CLA.Flag
+  ( flagB64
+  , flagJson
+  , flagVerify
+  , flagGroupCheck
+  ) where
+
+import Options.Applicative (Parser, flag, help, long, short)
+
+flagB64 :: Parser Bool
+flagB64 = flag False True $ mconcat
```

Comment:
While we are at it, you can refactor it to use [`switch`](https://hackage.haskell.org/package/optparse-applicative-0.16.1.0/docs/Options-Applicative-Builder.html#v:switch) instead of `flag False True` (but that's optional of course).